jenkins/Jenkinsfile

```diff
@@ -0,0 +1,33 @@
+@Library('pcic-pipeline-library')_
+
+
+node {
+    stage('Code Collection') {
+        collectCode()
+    }
+
+    stage('Testing') {
+        def requirements = ['requirements.txt', 'test_requirements.txt']
+        def pytestArgs = '-v --tb=short tests'
+        def options =  [containerData: 'crmprtd']
+
+        parallel "Python 3.6": {
+            runPythonTestSuite('pycds-test-python3.6', requirements, pytestArgs,
+                               options)
+        },
+        "Python 3.7": {
+            runPythonTestSuite('pycds-test-python3.7', requirements, pytestArgs,
+                               options)
+        }
+    }
+
+    if (isPypiPublishable()) {
+        stage('Push to PYPI') {
+            publishPythonPackage('pycds-test-python3.6', 'PCIC_PYPI_CREDS')
+        }
+    }
+
+    stage('Clean Workspace') {
+        cleanWs()
+    }
+}
```

Comment:
The clarity and elegance of this code shows you've factored out the common stages and steps well. Nice.

Reply:
Woo! :confetti_ball: